template/apps/web/src/pages/_app/PageConfig/OnboardingForm/components/CheckboxForm/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC, memo, useState } from 'react';
+import { Chip, Stack, Button, Title, Text, useMantineTheme } from '@mantine/core';
+
+import { useStyles } from './styles';
+
+interface CheckboxFormProps {
+  name: string;
+  options: string[];
+  title: string;
+  subtitle: string;
+  onSubmit: (name: string, value: string) => void;
+  onSkip: () => void;
+}
+
+const CheckboxForm: FC<CheckboxFormProps> = ({
+  name,
+  title,
+  subtitle,
+  options,
+  onSubmit,
+  onSkip,
+}) => {
+  const { classes } = useStyles();
+  const { colors } = useMantineTheme();
+
+  const [value, setValue] = useState('');
```

Comment:
let's use `useForm` hook and zod schema for each step of form.
It will be much more flexible for the complex forms